scripts/InstallPrerequisites-macOS.sh

```diff
@@ -36,7 +36,7 @@ tar -xvf spoa-v3.4.0.tar.gz
 # Build the static library.
 mkdir build-static
 cd build-static
-cmake ../spoa-v3.4.0 -DBUILD_SHARED_LIBS=OFF
+cmake ../spoa-v3.4.0 -DBUILD_SHARED_LIBS=OFF -Dspoa_optimize_for_native=OFF
```

Comment:
This way we lose this optimization for macOS 10, but since macOS 10 on its way out and the Shasta macOS version is provided for convenience rather than performance, I agree that we don't need to worry about this.